tests/manual-test-cases/Group15-Dogfood/15-3-OVA.robot

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 15-03 - Appliance and Network basic settings
+Resource  ../../resources/Util.robot
+Suite Setup  Setup Suite Environment
+Suite Teardown  Cleanup Suite Environment
+
+*** Variables ***
+${dhcp}  True
+${not_collect_log}  False
+${esx_number}  1
+${network}  False
+${maximum_retry}  10
+
+*** Keywords ***
+Setup Suite Environment
+    @{vms}=  Create A Simple VC Cluster  datacenter  cls  ${esx_number}  ${network}
+    ${vm_num}=  Evaluate  ${esx_number}+1
+    ${vm-names}=  Get Slice From List  ${vms}  0  ${vm_num}
+    ${vm-ips}=  Get Slice From List  ${vms}  ${vm_num}
+    Set Suite Variable  ${vc-ip}  @{vm-ips}[${esx_number}]
+    Set Suite Variable  ${vm-names}  ${vm-names}
+    Set Suite Variable  ${vm-ips}  ${vm-ips}
+    ${size}=  Evaluate  50*1024*1024
+    Set Suite Variable  ${ova_default_disk_size_inKB}  ${size}
+    ${size}=  Evaluate  80*1024*1024
+    Set Suite Variable  ${ova_adjusted_disk_size_inKB}  ${size}
+    GOVC Use VC Environment 
+
+Cleanup Suite Environment
+    Nimbus Cleanup  ${vm-names}  ${not_collect_log}
+
+Check VCH VM Disk Size 
+    [Arguments]  ${target_size}
+    ${size}=  Run  govc device.info -json -vm ${ova_target_vm_name} disk-1000-1 | jq .Devices[].CapacityInKB
+    
+    Should Be Equal As Integers  ${size}  ${target_size}
+
+Resize VCH VM Disk
+    [Arguments]  ${target_size}
+    Run  govc vm.disk.change -vm=${ova_target_vm_name} -disk.name "disk-1000-1" -size=${target_size}GB
+
+GOVC Use VC Environment
+    Set Environment Variable  GOVC_URL  ${vc-ip}
+    Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  GOVC_PASSWORD  Admin\!23
+
+GOVC Use ESXi Environment
+    Set Environment Variable  GOVC_URL  @{vm-ips}[0]
+    Set Environment Variable  GOVC_USERNAME  root
+    Set Environment Variable  GOVC_PASSWORD  ${NIMBUS_ESX_PASSWORD}
+
+*** Test Cases ***
+Check VCH VM Network Settings
+    [Setup]  Deploy VIC-OVA To Test Server
+    :FOR  ${IDX}  IN RANGE  ${maximum_retry}
```

Comment:
If you want to avoid that loop: `govc vm.ip -n ethernet-0 -v4`
That command will wait for property updates from VC/ESX until Get.Net is populated with an IpAddress for ethernet-0.